index.html

```diff
@@ -2,7 +2,1325 @@
+            </section>
+            <section class="header__info info-currency">
+                <div class="currency-course">
+                    <a href="#">Курс ЦБ</a>
+                </div>
+                <div class="currency-course-usd">
+                    <span>USD</span>
+                    <span>64.15</span>
+                </div>
+                <div class="currency-course-eur">
+                    <span>EUR</span>
+                    <span>68.47</span>
+                </div>
+                <a href="#" class="currency-option">Средневзвешенный</a>
+                <div class="purchase-selling">
+                    <span>Покупка</span>
+                    <span>Продажа</span>
+                </div>
+                <div class="purchase-selling-usd">
+                    <span>63.13</span>
+                    <span>65.08</span>
+                </div>
+                <div class="purchase-selling-eur">
+                    <span>66.88</span>
+                    <span>68.91</span>
+                </div>
+            </section>
+            <section class="header__info header-forms">
+                <form method="post" action="#" class="form-search">
+                    <label for="search">Поиск:</label>
+                    <input type="search" id="search" class="form-search-search">
+                    <button class="button-submit"></button>
+                </form>
+                <form method="post" action="#" class="form-auth">
+                    <label for="email">Почта:</label>
+                    <input type="email" id="email" class="form-auth-email" placeholder="Имя @e1.ru">
+                    <input type="password" id="pass" class="form-auth-pass">
+                    <button class="button-submit"></button>
+                </form>
+            </section>
+        </section>
+    </header>
+    <main class="content">
+        <section class="content__left">
+            <section class="content__main-news">
+                <figure class="content__main-news-content">
+                    <a href="#">
+                        <img src="img/main-news-img.jpg" alt="Главная новость" title="Главная новость">
+                    </a>
+                    <figcaption class="content__main-news-description">
+                        <span class="content__main-news-title">
+                            Вид изнутри: смотрим, как покрываются
+                            "золотом" купола в храме на Компрессорном
+                        </span>
+                        <span class="content__main-news-subtitle">
+                            До конца года их с помощью автокрана поднимут наверх.
+                        </span>
+                    </figcaption>
+                </figure>
+            </section>
+            <section class="content__place content__other-news">
+                <header class="content__header">
+                    <a href="#" class="content__header-title">Новости</a>
```

Comment:
В структуре документа не хватает заголовков которые h1…h6. Тэг `<header>` не в счет, потому что

> The HTML `<header>` element represents a group of introductory or navigational aids. It may contain some heading elements but also other elements like a logo, wrapped section's header, a search form, and so on.

Названия разделов прекрасно для этого подходят.